debian/distributionscanner_test.go

```diff
@@ -13,28 +14,40 @@ import (
 func TestDistributionScanner(t *testing.T) {
 	ver := regexp.MustCompile(`^\d+ \(\w+\)$`)
 	ctx := zlog.Test(context.Background(), t)
-	ents, err := os.ReadDir(`testdata/dist`)
+	ents, err := os.ReadDir(`testdata/distroless_dist`)
```

Comment:
We want to keep the `testdata/dist` here right? So we're still testing the normal debian os-release files

Reply:
My mistake, you're right! I've corrected it.